sdk/keyvault/azure-security-keyvault-keys/src/main/java/com/azure/security/keyvault/keys/cryptography/LocalKeyEncryptionKeyClient.java

```diff
@@ -0,0 +1,48 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.security.keyvault.keys.cryptography;
+
+import com.azure.core.cryptography.KeyEncryptionKey;
+
+/**
+ * A key client which is used to synchronously encrypt, or wrap, another key.
+ */
+public final class LocalKeyEncryptionKeyClient implements KeyEncryptionKey {
```

Comment:
Why doesn't this sync client extend from LocalCryptographyClient like the async client does?

Reply:
Follows the usual pattern, because it uses the async equivalent and blocks on it.
Extending the LocalCryptography client is not a requirement and doing so will require the LocalKeyEncryptionKeyClientBuilder to instantiate LocalCryptographyAsyncClient, which is not the right place for it.